src/LuaLib.ts

```diff
@@ -122,8 +128,9 @@ export function getLuaLibModulesInfo(emitHost: EmitHost): LuaLibModulesInfo {
     return luaLibModulesInfo;
 }
 
-export function readLuaLibFeature(feature: LuaLibFeature, emitHost: EmitHost): string {
-    const featurePath = path.resolve(__dirname, `../dist/lualib/${feature}.lua`);
+export function readLuaLibFeature(feature: LuaLibFeature, luaTarget: LuaTarget, emitHost: EmitHost): string {
+    const lualibDir = `../dist/${luaTarget === LuaTarget.Lua50 ? "lualib-lua50" : "lualib"}`;
```

Comment:
Duplicate code with line 119, also we should probably be using path.join, proposal:
```ts
function resolveLuaLibDir(target: LuaTarget) {
    const lualibDir = luaTarget === LuaTarget.Lua50 ? "lualib-lua50" : "lualib";
    return path.resolve(__dirname, "dist", lualibDir);
}
```

Reply:
Implemented with https://github.com/TypeScriptToLua/TypeScriptToLua/pull/1263/commits/e775d2b45557ed89c5f3a31c709f71d0e49b3ed4.

Reply:
Implemented with https://github.com/TypeScriptToLua/TypeScriptToLua/commit/28cc8cdd5be3ef523c9bc8905676fc02c51755e0.